src/redux/actions/samples/loadSamples.js

```diff
@@ -1,22 +1,79 @@
 import fetchAPI from 'utils/http/fetchAPI';
 import handleError from 'utils/http/handleError';
+import config from 'config';
+import { api } from 'utils/constants';
 
 import {
   SAMPLES_LOADED,
   SAMPLES_ERROR,
   SAMPLES_LOADING,
 } from 'redux/actionTypes/samples';
 
+const toApiV2 = (samples) => {
+  const apiV2Samples = [{ samples: {} }];
+
+  const buildApiv1FilesObject = (fileObject) => {
+    const fileNames = [];
+    const apiV1FileObject = {};
+
+    Object.keys(fileObject).forEach((key) => {
+      const fileName = fileObject[key].s3Path.split('/').pop();
```

Comment:
The name shouldn't be taken from the `s3Path`, in fact we don't really need to use s3Path anywhere in the UI.

The fileName can be guessed from the fileType (I do that translation in processUploadV2):
features10x -> 'features.tsv.gz'
barcodes10x -> 'barcodes.tsv.gz'
matrix10x -> 'matrix.mtx.gz'